engine/runtime-benchmarks/src/main/java/org/enso/interpreter/bench/benchmarks/semantic/VectorBenchmarks.java

```diff
@@ -38,10 +38,15 @@ public void initializeBenchmark(BenchmarkParams params) throws Exception {
         import Standard.Base.Data.Vector.Builder
         import Standard.Base.Data.Vector.Vector
         import Standard.Base.Data.Array_Proxy.Array_Proxy
+        from Standard.Base.Data.Boolean import False
 
         avg arr =
-            sum acc i = if i == arr.length then acc else
-                @Tail_Call sum (acc + arr.at i) i+1
+            sum acc i =
+                stop = i == arr.length
```

Comment:
What effect do these changes have?

Reply:
It is easier to analyze the IGV graphs. Particularly one can select line 45 and ask for _show all nodes related to this line_. Otherwise I wouldn't be able to generate [these two screenshots](https://github.com/enso-org/enso/pull/10079#issuecomment-2140642492):

![1ms case](https://github.com/enso-org/enso/assets/26887752/9dce031c-0b81-4550-bc93-6255bc4b8bce)
![8ms case](https://github.com/enso-org/enso/assets/26887752/d30c926f-7d8a-49e0-8b24-5960cf5fc8cc)